src/main/res/layout/activity_server_file_video.xml

```diff
@@ -31,17 +31,28 @@
         android:layout_height="wrap_content"
         android:layout_gravity="center" />
 
+
     <FrameLayout
         android:id="@+id/layout_content"
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
+
         <SurfaceView
             android:id="@+id/surface"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
             android:layout_gravity="center"
             android:fitsSystemWindows="false" />
+        <Button
+            android:id="@+id/button"
+            android:layout_width="20dp"
+            android:layout_height="20dp"
+            android:background="@drawable/rotation"
```

Comment:
`src/main/res/drawable-xxhdpi/exit_screen.png` has not been used anywhere in the code. Seems like it's extra and can be removed!